File: server-dotNet/Com.EnuoCms.Web/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using Com.EnuoCms.Model.Entities;
using Com.EnuoCms.IService;

/**
 * Created by zhanxiaoping 
 * zhanxp@me.com
 */
namespace Com.EnuoCms.Web.Controllers
{
    [RoutePrefix("")]
    public class HomeController : Controller
    {
		[Route]
        public ActionResult Index()
        {
            return View();
        }

        [Route("about")]
        public ActionResult About()
        {
            ViewBag.Message = "Your application description page.";
            return View();
        }

    }
}

File: server-spring-boot/enuocms-boot/src/main/java/com/enuocms/boot/controller/HomeController.java
package com.enuocms.boot.controller;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;

/**
 * Created by zhanxiaoping on 2017/8/30.
 * zhanxp@me.com
 */
@Controller
@RequestMapping("/")
public class HomeController {
    
    @RequestMapping("")
    public String index(Model model) {
    	return "/home/index";
	}

    @RequestMapping("about")
    public String about(Model model) {

        model.addAttribute("message","Your application description page.");

        return "/home/about";
    }
}
